x/auctions/keeper/genesis.go

```diff
@@ -0,0 +1,61 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package keeper
+
+import (
+	"fmt"
+
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/evmos/evmos/v19/utils"
+	"github.com/evmos/evmos/v19/x/auctions/types"
+)
+
+func InitGenesis(ctx sdk.Context, k Keeper, data types.GenesisState) {
```

Comment:
why do we need to have these functions in this file instead on the `x/auctions` dir like all the other modules?

Reply:
I believe we should be consistent in the repo 👍 one question regarding this, since the genesis init/export is using mainly keeper functions, why are not implemented as a keeper's methods? It is also the approach used in the Cosmos SDK.

Reply:
good question! Not sure why was this design decision.
If want to keep it inside the keeper package, add it as a keeper func please, e.i:

```
func (k Keeper) InitGenesis(ctx sdk.Context, data types.GenesisState) {
```